### PR TITLE
Feat/issue302

### PR DESCRIPTION
**Pull request by taiki1frsh**
Besides issue #302, I implemented `Reward` query which wasn't done.

### CODE REVIEW

File: x/ecosystem-incentive/keeper/incentive_unit_service.go [hunk 1]
```diff
@@ -37,6 +37,18 @@ func (k Keeper) Register(ctx sdk.Context, msg *types.MsgRegister) (*[]types.Subj
...
 	}
 
+	// operation related to IncentiveUnitIdsByAddr
+	// if exists already, add incentuve unit id in msg into data
+	// if not, newly create and set
+	for _, addr := range msg.SubjectAddrs {
+		incentiveUnitIdsByAddr := k.GetIncentiveUnitIdsByAddr(ctx, addr.AccAddress())
+		incentiveUnitIdsByAddr = incentiveUnitIdsByAddr.CreateOrUpdate(addr, msg.IncentiveUnitId)
+
+		if err := k.SetIncentiveUnitIdsByAddr(ctx, incentiveUnitIdsByAddr); err != nil {
```
Comment (jununifi) [inline x/ecosystem-incentive/keeper/incentive_unit_service.go:47]: What is the maximum incentive units count per address?

File: x/ecosystem-incentive/keeper/incentive_unit_service.go [hunk 2]
```diff
@@ -67,9 +92,30 @@ func (k Keeper) GetIncentiveUnit(ctx sdk.Context, id string) (types.IncentiveUni
 	return incentiveUnit, true
 }
 
+func (k Keeper) GetIncentiveUnitIdsByAddr(ctx sdk.Context, address sdk.AccAddress) types.IncentiveUnitIdsByAddr {
+	store := ctx.KVStore(k.storeKey)
+	prefixStore := prefix.NewStore(store, []byte(types.KeyPrefixIncentiveUnitIdsByAddr))
+
+	bz := prefixStore.Get(address)
+	if len(bz) == 0 {
```
Comment (jununifi) [inline x/ecosystem-incentive/keeper/incentive_unit_service.go:100]: I think `if bz == nil` would be the correct case.

File: x/ecosystem-incentive/keeper/incentive_unit_service.go (hunk 1 above)
Comment (taiki1frsh) [reply x/ecosystem-incentive/keeper/incentive_unit_service.go:47]: does that validation make sense practically?
Users can change the address to create a unit even if there's such limitation, can't they?
even so, you think it has to have such limitation?

File: x/ecosystem-incentive/keeper/incentive_unit_service.go (hunk 2 above)
Comment (taiki1frsh) [reply x/ecosystem-incentive/keeper/incentive_unit_service.go]: I modified.

**Review by jununifi — APPROVED** [review]

**Pull request merged**